packages/react-components/react-list/stories/package.json

```diff
@@ -18,5 +18,8 @@
     "@fluentui/scripts-storybook": "*",
     "@fluentui/eslint-plugin": "*",
     "@fluentui/scripts-tasks": "*"
+  },
+  "dependencies": {
```

Comment:
did you add this by hand ? there is no logic in the migration generator that would actually do this hmm

Reply:
nvm - found the bug in the generator